R/tidyverse.R

```diff
@@ -255,43 +254,30 @@ use_tidy_style <- function(strict = TRUE) {
 #'
 #' @examples
 #' \dontrun{
-#' ## active project, interval = "since the last release"
+#' ## active project, interval = since the last release
 #' use_tidy_thanks()
 #'
-#' ## active project, interval = "since a specific datetime"
+#' ## active project, interval = since a specific datetime
 #' use_tidy_thanks(from = "2018-02-24T00:13:45Z")
 #'
-#' ## r-lib/usethis, since a certain date
-#' use_tidy_thanks(owner = "r-lib", repo = "usethis", from = "2018-05-01")
+#' ## r-lib/usethis, inteval = since a certain date
+#' use_tidy_thanks("r-lib/usethis", from = "2018-05-01")
 #'
 #' ## r-lib/usethis, up to a specific release
-#' use_tidy_thanks(
-#'   owner = "r-lib", repo = "usethis",
-#'   from = NULL, to = "v1.3.0"
-#' )
+#' use_tidy_thanks("r-lib/usethis", from = NULL, to = "v1.3.0")
 #'
 #' ## r-lib/usethis, since a specific commit, up to a specific date
-#' use_tidy_thanks(
-#'   owner = "r-lib", repo = "usethis",
-#'   from = "08a560d", to = "2018-05-14"
-#' )
+#' use_tidy_thanks("r-lib/usethis", from = "08a560d", to = "2018-05-14")
 #' }
-use_tidy_thanks <- function(owner = NULL,
-                            repo = NULL,
-                            from = releases(owner = owner, repo = repo)[[1]],
+use_tidy_thanks <- function(repo_spec = github_repo_spec(),
+                            from = releases(repo_spec)[[1]],
```

Comment:
I dislike using unexported functions for the defaults. But also like revealing the default.

Reply:
Should we add this as an argument to `use_github_labels()` too?

Reply:
I don't think we need to add `repo_spec` to `use_github_labels()`. I makes sense to me that it works on the active project.

I think `use_tidy_thanks()` is different and pretty unique because you are likely to use it while in another project, e.g. composing a post in the tidyverse.org project.